Makefile

```diff
@@ -96,6 +96,11 @@ beats-dashboards:
 	mkdir -p build/dashboards
 	$(foreach var,$(BEATS),cp -r $(var)/_meta/kibana/ build/dashboards/$(var)  || exit 1;)
 
+# Collects all dashboards and generates dashboard folder for https://github.com/elastic/beats-dashboards/tree/master/dashboards
+.PHONY: import-dashboards
+import-dashboards: beats-dashboards
```

Comment:
What is the use case of this? The main reason of import-dashboards is when working on a beat and quickly want to import the dashboards. When do you want to import the dashboards for all beats at the same time?

The other question would be why the `beats-dashboards` dependency is needed? If we need this I would expect it to depend on `update`